New Docs

PLOKMJNB opened the pull request.

PLOKMJNB committed:
new docs

PLOKMJNB committed:
Update README.md

PLOKMJNB committed:
maybe fix conflict?

PLOKMJNB committed:
Update README.md

PLOKMJNB committed:
Update README.md

PLOKMJNB committed:
edit txAdmin html doc links

aidenzybell commented on README.md:
```diff
@@ -9,7 +9,7 @@
 	</h4>
 	<p align="center">
-		<b>txAdmin</b> is a <b>full featured</b> web panel to Manage & Monitor your FiveM Server remotely, in use by over three thousand servers worldwide at any given time.
+		<b>txAdmin</b> is a <b>full featured</b> web panel to Manage & Monitor your FiveM/RedM Server remotely, in use by over fivem thousand servers worldwide at any given time.
```
Typo: "in use by over fivem thousand servers"
Should be "in use by over FIVE thousand servers"

PLOKMJNB replied:
https://github.com/tabarra/txAdmin/pull/316/commits/71865575397346b9a88979afa0e2057b406b42d3

PLOKMJNB committed:
typo

https://github.com/tabarra/txAdmin/pull/316/commits/41cb0ca2ce56e68f4e734287c35b8c32febbafbe#r602802305

PLOKMJNB committed:
Delete settings.json

PLOKMJNB committed:
Delete README.md

TasoOneAsia commented:
Will close for now pending whenever we want to reopen this to talk about
